README.md

```diff
@@ -5,15 +5,64 @@
 The explorer is deployed at https://cosmwasm.github.io/code-explorer and is configured
 for [Heldernet](https://gist.github.com/webmaster128/6385a7e1885211d9937ada2688ce34c3).
 
-## Run locally
+## Use local
 
-The node URL is configured in `src/settings.ts` and defaults to `http://localhost:1317` for development builds.
+You need to install the code-explorer dependencies before choosing which network you want to connect to:
 
 ```sh
 yarn install
-yarn start
 ```
 
+### Run against local networks
+
+Clone the CosmJS repo in order to run a local Launchpad or Stargate network:
+
+```sh
+git clone --depth 1 --branch v0.24.0-alpha.11 git@github.com:cosmos/cosmjs.git
```

Comment:
Let's use the HTTP connection, which is easier for guests: https://github.com/cosmos/cosmjs.git